beetsplug/info.py

```diff
@@ -78,14 +84,36 @@ def update_summary(summary, tags):
     return summary
 
 
-def print_data(data):
+def print_data(data, fmt=None, human_length=True):
+    """Print, with optional formatting, the fields of a single item.
+
+    If no format string `fmt` is passed, the entries on `data` are printed one
+    in each line, with the format 'field: value'. If `fmt` is not `None`, the
+    item is printed according to `fmt`, using the `Item.__format__` machinery.
+
+    If `raw_length` is `True`, the `length` field is displayed using its raw
+    value (float with the number of seconds and miliseconds). If not, a human
+    readable form is displayed instead (mm:ss).
+    """
+    item = data.pop('item', None)
+    if fmt:
+        # use fmt specified by the user, prettifying length if needed
+        if human_length and '$length' in fmt:
```

Comment:
This is a bit hackish, but does the trick. I'm not sure if there is already a better mechanism in place for converting a length to a "human-friendly" length in a format string somewhere in the codebase? If not, I'm wondering if it would be worth to try to come up with a cleaner solution and make the human-friendly length as a default for other functions as well.